### PR TITLE
data lifecycle doc updates

### LINKED ISSUE #3016: update data lifecycle docs
do the things.

- [ ] change page name from node lifecycle to data lifecycle
- [ ] add redirect (maybe?)
- [ ] make it clear whats set on a new install
- [ ] make it clear you need to use the save changes button to save changes
- [ ] add all the things
- [ ] new screenshot
- [ ] ping mary for review
- [ ] move relevant content from data lifecycle to node lifecycle
- [ ] prolly delete the data lifecycle page
- [ ] link to relevant api docs

### PR DESCRIPTION
**Pull request by susanev**
### :nut_and_bolt: Description: What code changed, and why?
hi, hi. we are close to merging the updates to the data lifecycle (previously called node lifecycle) page, so these are the associated docs updates.

this work should merge into the same release as #3158 

note this keeps the static api docs, but once those are added to the api docs then they can be removed, see #3015 

there are some api doc annotations that still need to be udpated

this work is unfinished and is being passed to @mjingle for completion 

### :chains: Related Resources
#1208 
fixes #3016 

### :+1: Definition of Done
- [x] add UI content to data lifecycle page
- [x] add a UI screenshot
- [x] move data lifecycle page to settings area
- [x] remove node lifecycle page
- [x] update other pages that reference the node lifecycle page with links or words
- [x] delete unused img: `/images/docs/node-lifecycle.png`
- [ ] ~update api doc annotations to use data lifecycle instead of node lifecycle~
- [x] docs review + editing

### :athletic_shoe: How to Build and Test the Change
`make serve` from `components/automate-chef-io`

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by susanev** [comment]
this is failing hugo lint, and there's not enough information in the output for me to know why. also, how do yall run hugo lint locally? happy to help troubleshoot if needed, let me know.

File: components/automate-chef-io/content/docs/data-lifecycle.md [hunk 1]
```diff
@@ -1,39 +1,66 @@
 +++
-title = "Data Lifecycle"
+title = "Data Lifeycle"
```
Comment (susanev) [inline components/automate-chef-io/content/docs/data-lifecycle.md]: oops

Comment (susanev) [inline components/automate-chef-io/content/docs/data-lifecycle.md]:
```suggestion
title = "Data Lifecycle"
```

File: components/automate-chef-io/content/docs/data-lifecycle.md [hunk 2]
```diff
@@ -1,39 +1,66 @@
...
+Note: Chef Automate data retention processes changed in 20191129172405. The [upgrade documentation]({{< ref "install/#upgrades" >}}) covers configuring your system to install new Chef Automate versions. For guidance, contact your customer support agent. You can also use the [previous data retention documentation](https://github.com/chef/automate/blob/20191104205453/components/automate-chef-io/content/docs/configuration.md#data-retention) for help with configuring data retention on older Chef Automate installations.
+{{% /warning %}}
+
+## Data Lifecycle UI


+
+Navigate to _Settings_ > _Data Lifecycle_ and adjust any settings you would like to change. After making changes use the **Save Changes** button to apply your changes.
+
+Only userse with `dataLifecycle:*` IAM access are able to see the data lifecycle job statuses, configure jobs, or run jobs.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/data-lifecycle.md]:
```suggestion
Only users with `dataLifecycle:*` IAM access are able to see the data lifecycle job statuses, configure jobs, or run jobs.
```

File: components/automate-chef-io/content/docs/applications-dashboard.md [hunk 3]
```diff
@@ -103,9 +103,7 @@ See the [Habitat CLI documentation](https://www.habitat.sh/docs/habitat-cli/#hab
 
 ### Marking a Service as Disconnected
 
-Configure the threshold for marking a service as disconnected through either Chef Automate or the Chef Automate API.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/applications-dashboard.md]: im not sure why these were added, this is not possible from the UI yet

File: components/automate-chef-io/content/docs/applications-dashboard.md [hunk 4]
```diff
@@ -124,9 +122,6 @@ The removal threshold is optional. The default is 7 days. Disabling the threshol
 
 Configure the threshold  for removing a disconnected service through either Chef Automate or the Chef Automate API. The threshold is the amount of time from when a service was marked as disconnected.
 
-To access the configuration in Chef Automate, navigate to the **Settings** tab, then select _Node Lifecycle_ on the sidebar.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/applications-dashboard.md]: im not sure why these were added, this is not possible from the UI yet

File: components/automate-chef-io/content/docs/applications-dashboard.md (hunk 3 above)
Comment (kagarmoe) [reply components/automate-chef-io/content/docs/applications-dashboard.md]: Weird. We must have thought it worked.

**Comment by susanev** [comment]
i added a do not merge label because this should not merge until #3158 merges

**Comment by susanev** [comment]
@mjingle i expect the data lifecycle ui to merge tomorrow, will it be possible to get these in before mondays promotion?

**Comment by susanev** [comment]
i removed the do not merge label because the implementation has been merged

File: components/automate-chef-io/content/docs/client-runs.md [hunk 5]
```diff
@@ -189,19 +205,20 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/missing-nodes/config
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or [use an Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests. 
+You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or [use an Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/client-runs.md:209]:
```suggestion

```

File: components/automate-chef-io/content/docs/client-runs.md [hunk 6]
```diff
@@ -216,4 +233,5 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/delete-nodes/config"
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or use a [Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/client-runs.md:236]:
```suggestion
You will need an [API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) to send API requests. 

```

we need to fix the link, but up to yall if you want to keep this a1 content.. seems like we shouldn't though?

File: components/automate-chef-io/content/docs/client-runs.md [hunk 7]
```diff
@@ -216,4 +233,5 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/delete-nodes/config"
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or use a [Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests. 
+You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or use a [Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/client-runs.md:237]:
```suggestion
```

File: components/automate-chef-io/content/docs/client-runs.md [hunk 8]
```diff
@@ -189,19 +205,20 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/missing-nodes/config
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or [use an Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/client-runs.md:208]:
```suggestion
You will need an [API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) to send API requests. 

```

we need to fix the link, but up to yall if you want to keep this a1 content.. seems like we shouldn't though?

File: components/automate-chef-io/content/docs/data-lifecycle.md [hunk 9]
```diff
@@ -217,23 +217,23 @@ curl -s -H "api-token: $TOKEN" -X PUT --data "@config.json" https://{{< example_
 All jobs have the following options:
 
 * `recurrence` (string) - A recurrence rule that determines how often, at what interval, and when to initially start a scheduled job. Any valid recurrence rule [as defined in section 4.3.10 of RFC 2445](https://www.ietf.org/rfc/rfc2445.txt) is valid in this field.
-* `disabled` (bool) - Whether or not this job should be enabled
+* `disabled` (bool) - True or false if this job is an enabled job.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/data-lifecycle.md:220]:
```suggestion
* `disabled` (bool) - True or false if this job is enabled.
```

File: components/automate-chef-io/content/docs/data-lifecycle.md [hunk 10]
```diff
@@ -217,23 +217,23 @@ curl -s -H "api-token: $TOKEN" -X PUT --data "@config.json" https://{{< example_
...
 Purge jobs have the following options:
 
-* `purge_polices` (map) - Configures how old the corresponding data must be in the configured storage backend before it is purged.
-  * `elasticsearch` (array) - An array of ElasticSearch purge policies
-    * `disabled` (bool) - Whether or not this job should be enabled
-    * `policy_name` (string) - The name of the purge policy you wish to update
-    * `older_than_days` (int) - The name of the purge policy you wish to update
+* `purge_polices` (map) - Configures how old the corresponding data must be in the configured storage backend before purging occurs.
+  * `elasticsearch` (array) - An array of Elasticsearch purge policies
+    * `disabled` (bool) - True or false if this job is an enabled job.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/data-lifecycle.md:230]:
```suggestion
    * `disabled` (bool) - True or false if this job is enabled.
```

File: components/automate-chef-io/content/docs/client-runs.md (hunk 7 above)
Comment (susanev) [reply components/automate-chef-io/content/docs/client-runs.md:237]: seems like we shouldnt be keeping this a1 content around, i added the corrected link to the line above

File: components/automate-chef-io/content/docs/client-runs.md (hunk 5 above)
Comment (susanev) [reply components/automate-chef-io/content/docs/client-runs.md:209]: seems like we shouldnt be keeping this a1 content around, i added the corrected link to the line above

Comment (mjingle) [reply components/automate-chef-io/content/docs/client-runs.md:209]: I do not see a different link in your suggested change? I understand about the Chef Automate 1 removal. So this sentence needs to be modified to only mention creating a new token?

Comment (susanev) [reply components/automate-chef-io/content/docs/client-runs.md:209]: this was being used, that does not work anymore
#creating-a-standard-api-token

this is the correct one
#creating-api-tokens

Comment (mjingle) [reply components/automate-chef-io/content/docs/client-runs.md:209]: I'm following more now. I was really thrown by the link not being included the correction, which you did in the other suggestions below. Incorporating

File: components/automate-chef-io/content/docs/data-lifecycle.md (hunk 9 above)
Comment (mjingle) [reply components/automate-chef-io/content/docs/data-lifecycle.md:220]: I'll allow the passive tense in this case. I was trying to avoid using "is."

Comment (susanev) [reply components/automate-chef-io/content/docs/data-lifecycle.md:220]: oh i see, i found the original a bit difficult to parse

Comment (mjingle) [reply components/automate-chef-io/content/docs/data-lifecycle.md:220]: You are right that it's hard to parse. I'm changing it back regardless :)
